templates/search.html

```diff
@@ -8,6 +8,15 @@ <h1>Search</h1>
       <main role="main" class="inner cover">
         <input class="form-control" id="bookInput" type="text" placeholder="Search...">
         <ul class="list-group" id="bookList">
+            <li class="list-group-item my_container my_container_header">
+                <div class="col-lg-6">
+                    <font color="white"><b>Title</b></font>
+                </div>
+                <div class="col-lg-6">
+                    <font color="white"><b>Author</b></font>
```

Comment:
I don't know if it would not be better to move define font color to CSS style sheet :)

Reply:
done